### ISSUE TITLE
Error to build project after installation of this package

### ISSUE THREAD

**Issue by teckbeng-payboy** [alpha0010/react-native-file-access#44]
Version
    "react": "17.0.2",
    "react-native": "0.67.2",

**Comment by alpha0010** [comment]
What is the error you are seeing?

Does the example project work for you?
```sh
git clone https://github.com/alpha0010/react-native-file-access.git
cd react-native-file-access
yarn
yarn example android
yarn example ios
```

**Comment by Navipro70** [comment]
@alpha0010
Compiling for iOS 10.0, but module 'ZIPFoundation' has a minimum deployment target of iOS 12.0

I thinks needs up deployment target to 12.

**Comment by honzajerabek** [comment]
I have the same problem, cocoapods automatically resolved [ZIPFoundation 0.9.13](https://github.com/weichsel/ZIPFoundation/blob/0.9.13/ZIPFoundation.podspec), which is minimum iOS `12.0` but `react-native-file-access` has minimum iOS `11.0`.

This leads to build error.

I was able to workaround by manually specifing the older version that @alpha0010  mentioned in my Podfile:

```
...

pod 'ZIPFoundation', '0.9.11'

...
```
It doesn't seem right though

**Comment by alpha0010** [comment]
Should be fixed in v2.2.2.
